sys-fs/dislocker/files/dislocker-0.7.3_slotted-mbedtls3.patch

```diff
@@ -0,0 +1,26 @@
+https://bugs.gentoo.org/804981
```

Comment:
Was this sent upstream? If not, please send it. If it was, please include the link to the PR at the top of the patch (always).